examples/src/index.tsx

```diff
@@ -243,7 +245,21 @@ storiesOf('FlowMapLayer', module)
         </LegendBox>
       </>
     )),
-  );
+  )
+  .add('custom legend', () => (
+    <>
+      <LegendBox bottom={35} left={10}>
+        <DiffColorsLegend positiveText="+ diff" negativeText="- diff" />
+        <hr />
+        <LocationTotalsLegend
+          diff={true}
+          aboutEqualText="unequal"
```

Comment:
Could you change the values of the `xxText` props in the story too? This way they are confusing.